### PR TITLE
Cleanup ANU, Sydney, Melbourne, UTS

### PR DESCRIPTION
**Pull request by hosking**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [X] All pull requests and issues must come from non-anonymous accounts.

- [X] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [ ] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

**Comment by emeryberger** [comment]
```
Sanity checking the commit. Please check any issues raised here.
...
Processing Sanjay Chawla,Hamad Bin Khalifa University,https://www.hbku.edu.qa/en/staff/dr-sanjay-chawla,fdUJcwYAAAAJ
  ERROR: This institution (Hamad Bin Khalifa University) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3ASanjay%20Chawla:$%3A&format=json&c=10
  Checking homepage URL (https://www.hbku.edu.qa/en/staff/dr-sanjay-chawla)
***Test failure for Sanjay Chawla***.
Processing Zhengfeng Ji,Tsinghua Universuty,https://www.cs.tsinghua.edu.cn/info/1[12](https://github.com/emeryberger/CSrankings/actions/runs/5618606865/job/15224453515?pr=6098#step:6:13)1/4990.htm,2uXdu7AAAAAJ
  ERROR: This institution (Tsinghua Universuty) was not found in `institutions.csv`.
  Checking https://dblp.org/search/author/api?q=author%3AZhengfeng%20Ji:$%3A&format=json&c=10
  WARNING: Possibly invalid name (Zhengfeng Ji). This may be a disambiguation entry.
  Checking homepage URL (https://www.cs.tsinghua.edu.cn/info/1121/4990.htm)
***Test failure for Zhengfeng Ji***.
```

In addition, Steve Blackburn is AFAICT on >= 25% leave in industry and thus is not eligible for inclusion.

**Comment by hosking** [comment]
Have resolved issues noted.

Steve Blackburn continues to maintain a full complement of PhD students at ANU, but I note <25% employment disqualifies him. OTOH: there are a number of folks at other ANU universities whom I plan to challenge on similar grounds (viz Anton van den Hengel at Adelaide who is <5% employed there as he is full-time at Amazon). Expect more merge requests to come.

> ```
> Sanity checking the commit. Please check any issues raised here.
> ...
> Processing Sanjay Chawla,Hamad Bin Khalifa University,https://www.hbku.edu.qa/en/staff/dr-sanjay-chawla,fdUJcwYAAAAJ
>   ERROR: This institution (Hamad Bin Khalifa University) was not found in `institutions.csv`.
>   Checking https://dblp.org/search/author/api?q=author%3ASanjay%20Chawla:$%3A&format=json&c=10
>   Checking homepage URL (https://www.hbku.edu.qa/en/staff/dr-sanjay-chawla)
> ***Test failure for Sanjay Chawla***.
> Processing Zhengfeng Ji,Tsinghua Universuty,https://www.cs.tsinghua.edu.cn/info/1[12](https://github.com/emeryberger/CSrankings/actions/runs/5618606865/job/15224453515?pr=6098#step:6:13)1/4990.htm,2uXdu7AAAAAJ
>   ERROR: This institution (Tsinghua Universuty) was not found in `institutions.csv`.
>   Checking https://dblp.org/search/author/api?q=author%3AZhengfeng%20Ji:$%3A&format=json&c=10
>   WARNING: Possibly invalid name (Zhengfeng Ji). This may be a disambiguation entry.
>   Checking homepage URL (https://www.cs.tsinghua.edu.cn/info/1121/4990.htm)
> ***Test failure for Zhengfeng Ji***.
> ```
> 
> 
>     
>   
> 
> In addition, Steve Blackburn is AFAICT on >= 25% leave in industry and thus is not eligible for inclusion.

**Comment by hosking** [comment]
> Have resolved issues noted.
> 
> Steve Blackburn continues to maintain a full complement of PhD students at ANU, but I note <75% employment disqualifies him. OTOH: there are a number of folks at other AUS universities whom I plan to challenge on similar grounds (viz Anton van den Hengel at Adelaide who is <5% employed there as he is full-time at Amazon). Expect more merge requests to come.
> 
> > ```
> > Sanity checking the commit. Please check any issues raised here.
> > ...
> > Processing Sanjay Chawla,Hamad Bin Khalifa University,https://www.hbku.edu.qa/en/staff/dr-sanjay-chawla,fdUJcwYAAAAJ
> >   ERROR: This institution (Hamad Bin Khalifa University) was not found in `institutions.csv`.
> >   Checking https://dblp.org/search/author/api?q=author%3ASanjay%20Chawla:$%3A&format=json&c=10
> >   Checking homepage URL (https://www.hbku.edu.qa/en/staff/dr-sanjay-chawla)
> > ***Test failure for Sanjay Chawla***.
> > Processing Zhengfeng Ji,Tsinghua Universuty,https://www.cs.tsinghua.edu.cn/info/1[12](https://github.com/emeryberger/CSrankings/actions/runs/5618606865/job/15224453515?pr=6098#step:6:13)1/4990.htm,2uXdu7AAAAAJ
> >   ERROR: This institution (Tsinghua Universuty) was not found in `institutions.csv`.
> >   Checking https://dblp.org/search/author/api?q=author%3AZhengfeng%20Ji:$%3A&format=json&c=10
> >   WARNING: Possibly invalid name (Zhengfeng Ji). This may be a disambiguation entry.
> >   Checking homepage URL (https://www.cs.tsinghua.edu.cn/info/1121/4990.htm)
> > ***Test failure for Zhengfeng Ji***.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > In addition, Steve Blackburn is AFAICT on >= 25% leave in industry and thus is not eligible for inclusion.

Meant to say "at other AUS universities".

**Comment by emeryberger** [comment]
```
Processing Hong Shen 0001,Sun Yat-Sen University,https://ieeexplore.ieee.org/author/37279492700,tBnVrjEAAAAJ
  ERROR: This institution (Sun Yat-Sen University) was not found in `institutions.csv`.
```

(needs to be "Sun Yat-sen University", note case)

 
```
Processing Zhengfeng Ji,Tsinghua University,https://www.cs.tsinghua.edu.cn/info/1121/4990.htm,2uXdu7AAAAAJ
  Checking https://dblp.org/search/author/api?q=author%3AZhengfeng%20Ji:$%3A&format=json&c=10
  WARNING: Possibly invalid name (Zhengfeng Ji). This may be a disambiguation entry.
  Checking homepage URL (https://www.cs.tsinghua.edu.cn/info/1121/4990.htm)
***Test failure for Zhengfeng Ji***.
```

Above is fine, but also needs alias entry for "Zheng-Feng Ji".

**Comment by hosking** [comment]
> ```
> Processing Hong Shen 0001,Sun Yat-Sen University,https://ieeexplore.ieee.org/author/37279492700,tBnVrjEAAAAJ
>   ERROR: This institution (Sun Yat-Sen University) was not found in `institutions.csv`.
> ```
> 
> 
>     
>   
> 
> (needs to be "Sun Yat-sen University", note case)
> 
> 
> ```
> Processing Zhengfeng Ji,Tsinghua University,https://www.cs.tsinghua.edu.cn/info/1121/4990.htm,2uXdu7AAAAAJ
>   Checking https://dblp.org/search/author/api?q=author%3AZhengfeng%20Ji:$%3A&format=json&c=10
>   WARNING: Possibly invalid name (Zhengfeng Ji). This may be a disambiguation entry.
>   Checking homepage URL (https://www.cs.tsinghua.edu.cn/info/1121/4990.htm)
> ***Test failure for Zhengfeng Ji***.
> ```
> 
> 
>     
>   
> 
> Above is fine, but also needs alias entry for "Zheng-Feng Ji".

Done

File: csrankings-r.csv [hunk 1]
```diff
@@ -1365,7 +1365,7 @@ Rui Wang 0004,Zhejiang University,http://www.cad.zju.edu.cn/home/rwang,yUxnN_EAA
 Rui Wang 0015,Shanghai Jiao Tong University,http://bcmi.sjtu.edu.cn/home/wangrui,oTU0v5IAAAAJ
 Rui Yamaguchi,University of Tokyo,http://bonsai.hgc.jp/~ruiy,NOSCHOLARPAGE
 Rui Yan 0001,Renmin University of China,http://www.ruiyan.me,NOSCHOLARPAGE
-Rui Zhang 0003,University of Melbourne,https://people.cs.umass.edu/~ruiwang,yoJqPIkAAAAJ
+Rui Zhang 0003,Tsinghua University,https://www.ruizhang.info,yoJqPIkAAAAJ
```
Comment (emeryberger) [inline csrankings-r.csv:1368]: Visiting professors are not eligible for inclusion. I will remove.

**Pull request merged**